website/docs/artist_hosts_maya_yeti.md

```diff
@@ -44,34 +46,51 @@ You can now publish Yeti cache as any other types. **OpenPype → Publish**. It
 create sequence of `.fur` files and `.fursettings` metadata file with Yeti node
 setting.
 
-### Loading Yeti caches
+:::note Collect Yeti Cache failure
+If you encounter **Collect Yeti Cache** failure during collecting phase, and the error is like
+```fix
+No object matches name: pgYetiMaya1Shape.cbId
+```
+then it is probably caused by scene not being saved before publishing.
+:::
+
+### Loading
 
 You can load Yeti cache by **OpenPype → Load ...**. Select your cache, right+click on
 it and select **Load Yeti cache**. This will create Yeti node in scene and set its
 cache path to point to your published cache files. Note that this Yeti node will
 be named with same name as the one you've used to publish cache. Also notice that
 when you open graph on this Yeti node, all nodes are as they were in publishing node.
 
-### Creating and publishing Yeti Rig
+## Yeti Rigs
+
+### Creating and publishing
+
+Yeti Rigs are designed to connect to published models or animation rig. The workflow gives the Yeti Rig full control on that geometry to do additional things on top of whatever input comes in, e.g. deleting faces, pushing faces in/out, subdividing, etc.
+
+Let's start with a [model](artist_hosts_maya.md#loading-model) or [rig](artist_hosts_maya.md#loading-rigs) loaded into the scene. Here we are using a simple rig.
+
+![Maya - Yeti Simple Rig](assets/maya-yeti_simple_rig.png)
+
+We'll need to prepare the scene a bit. We want some Yeti hair on the ball geometry, so duplicating the geometry, adding the Yeti hair and grouping it together.
 
-Yeti Rigs are working in similar way as caches, but are more complex and they deal with
-other data used by Yeti, like geometry and textures.
+![Maya - Yeti Hair Setup](assets/maya-yeti_hair_setup.png)
 
-Let's start by [loading](artist_hosts_maya.md#loading-model) into new scene some model.
-I've loaded my Buddha model.
+Now we need to connect the Yeti Rig with the animation rig. Yeti Rigs work by publishing the attribute connections from its input nodes and reconnect them later in the pipeline. This means we can only use attribute connections to get the animation from the rig to the Yeti Rig. Constraints or blendshapes are not supported. We'll connnect the Yeti Rig geometry to the animation rig, with the transform and mesh attributes.
```

Comment:
When reading this my head goes "oh, boy.. that's limiting!" but in practice you can do whatever you want with constraints and blendshapes - those connections however would be internal to your rig.

So say you'd want to use an external mesh as blendshape in your rig you'd do:
- `.outMesh` -> `.inMesh` connection or alike from OUTSIDE the rig into a copy of the mesh INSIDE the rig.
- Then from that internal mesh you can do blendshapes just fine, etc.

I'm wondering whether we'd need to somehow elaborate on how that could work. I'm a tiny bit worried that someone starts to think they can't use e.g. blendshapes inside the yeti rig whatsoever.

Reply:
Will elaborate on this.